README.md

```diff
@@ -50,13 +59,15 @@ So there are a few tasks `semantic-git-release-cli` will do for you:
 - creates or updates the `CHANGELOG.md`
 - commits and tags the new `version`
 
-## Recover the CHANGELOG.md
+### Recover the complete changelog
 
 If you are start to use `semantic-git-release-cli` but already released (and tagged) versions. Simply use the recover mode.
 
+If `--recover` is set to `backup` it will generate a backup file of your current `CHANGELOG.md` inside of the `.sgr_backup` directory, which gets generated the first time you use this command inside of your current working directory.
+
 ```sh
-# generates the complete CHANGELOG.md
-sgr --recover # or short `sgr -r`
-# generates the complete CHANGELOG.md and creates a backup of the current CHANGELOG.md in .sgr_backup
-sgr -r backup # or short `sgr -r b`
+# generates the complete CHANGELOG.md without backup
+$ sgr --recover # or short `sgr -r`
+# generates the complete CHANGELOG.md with backup
+$ sgr -r backup # or short `sgr -r b`
```

Comment:
Maybe write `$sgr --recover backup`, as you write the short flag anyway in the comment.